consensus/node.go

```diff
@@ -16,6 +16,12 @@ import (
 	tmtServer "github.com/tendermint/tendermint/abci/server"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/lightstreams-network/lightchain/consensus/metrics"
+	"github.com/tendermint/tendermint/version"
+)
+
+var (
+	siriusProtocolBlockVersion version.Protocol = 8
+	standaloneProtocolBlockVersion = version.BlockProtocol
```

Comment:
Standalone uses the latest protocol version by default

Reply:
what's the advantage of having 2 different protocol versions? Why locally working with one and then on test network on the other one? Shouldn't they be the same?

Reply:
There is not actual different as Tendermint do not have separate execution depending on the version. But I think it makes sense to use the latest version on which the blockchain was initialized

Reply:
ok, so nothing changes between the versions then? The mainnet will be the same like standalone then (new) one right? Should we migrate the sirius to latest one somehow? I am asking because I am worried there is a difference somewhere and we are testing on one, but then mainnet will be on other.

Reply:
Created a new task: https://github.com/lightstreams-network/lightchain/issues/87, resolving this for the moment being